### ISSUE TITLE
How to open links of wikipedia in new tab by default or by ctrl+click as is possible in web browsers?

### ISSUE THREAD

**Issue by AntiMortal** [gnosygnu/xowa#762]
I mean in xowa app. I tried making various shortcuts but somehow they just open empty tab and nothing happens.
Is there any way? I would prefer to make it default except for search suggestion links. Meaning if I click on any underlned link that it opens in new tab, but maybe even those in search box. Or regardless of search box is there any way?

**Comment by gnosygnu** [comment]
> by ctrl+click as is possible in web browsers? 
> Meaning if I click on any underlned link that it opens in new tab, but maybe even those in search box. Or regardless of search box is there any way?

Hey, sorry for the late reply. Been meaning to comment, but busy lately

XOWA supports "Middle Click" for links out of box.  Doing "Ctrl + Left Click" is not supported though. Do you only have a 2 button mouse? If so, I can look into adding "Ctrl + Left Click" functionality (I didn't know that was a function until now)

Also, in case you're interested, the options page for shortcuts is `home/wiki/Special:XowaCfg?grp=xowa.gui.shortcuts`. For opening pages in new tab it would be `Open Link in New Tab`. However, `Ctrl + Left Click` doesn't work as XOWA doesn't listen for combination of keyboard + mouse events

**Comment by AntiMortal** [comment]
Yes I mean like in web browsers chrome and firefox. Also can you please allow default option to open all links in new tab and follow?
I have wheel mouse too and can also reconfigure elan touchpad to use left click as mouse middle click. But if there could be option to follow too, not just open in new tab. Now I am using server and firefox as solution. But best would be if you allow us to open by default in new tab + follow.
